Course/Course.xcodeproj/project.pbxproj

```diff
@@ -526,13 +525,20 @@
 			path = Mock;
 			sourceTree = "<group>";
 		};
-		97CA95212B875EA200A9EDEA /* Views */ = {
+		9784D4752BF39EEF00AFEFFF /* CalendarSyncProgressView */ = {
```

Comment:
Any reason to update this? Asking because it doesn't seem relevant to the work.

Reply:
**Updated location from:**
```
Course/Views/CalendarSyncProgressView.swift
Course/Views/DatesSuccessView.swift
```
**To:**
```
Course/Presentation/Subviews/CalendarSyncProgressView/CalendarSyncProgressView.swift
Course/Presentation/Subviews/DatesSuccessView/DatesSuccessView.swift
```

I found that we already have subviews folder and it seems more appropriate to place it there.